domain/ide/command/code_fix_feedback.go

```diff
@@ -38,8 +38,8 @@ func (cmd *codeFixFeedback) Command() types.CommandData {
 func (cmd *codeFixFeedback) Execute(ctx context.Context) (any, error) {
 	args := cmd.command.Arguments
 	fixId := args[0].(string)
-	positive := args[1].(bool)
-	err := cmd.apiClient.SubmitAutofixFeedback(ctx, fixId, positive)
+	feedback := args[1].(string)
+	err := cmd.apiClient.SubmitAutofixFeedback(ctx, fixId, feedback)
```

Comment:
Let's change this to invoke a go routine.

Reply:
Also, let's also analyse, if this needs to be added and respected when using globalIgnoresFlow and code-client-go

Reply:
Go routine fixed.

Reply:
So, I have been looking through code-client-go and Go Application Framework, trying to find anything indicating that they support autofix/ai fix, and that I would need to add support for ai fix feedback. But I could not find anything on autofix.

I also did code scans with ignores enabled, towards dev16, and applied AI fixes. I successfully saved feedback for the issues.